bindings/tests/pravega_reader_test.py

```diff
@@ -92,18 +96,18 @@ def test_multipleReader(self):
         r1.reader_offline()
 
         r2 = reader_group.create_reader("r2")
-        slice2 = _run(self.get_segment_slice(r2))
+        slice2 = await r2.get_segment_slice_async()
         for event in slice2:
             count+=1
             self.assertEqual(b'data', event.data(), "Invalid event data")
 
         print("Number of events read after consuming slice2 ", count)
         self.assertEqual(count, 100, "100 events are expected")
 
-    # This test verifies data reading a Pravega stream with multiple readers.
-    # It also invokes release Segment after consuming part of the first segment slice and marks the first
-    # reader as offline. This test verifies if we are able to read all the 100 elements.
-    def test_multipleReaderPartialRead(self):
+    # # This test verifies data reading a Pravega stream with multiple readers.
```

Comment:
I guess the `#` is duplicate here and the next two lines?

Reply:
fixed it.